cluster/addons/fluentd-elasticsearch/es-image/Dockerfile

```diff
@@ -17,7 +17,7 @@ FROM docker.elastic.co/elasticsearch/elasticsearch-oss:6.3.2
 VOLUME ["/data"]
 EXPOSE 9200 9300
 
-COPY elasticsearch_logging_discovery run.sh bin/
+COPY elasticsearch_logging_discovery.go run.sh bin/
```

Comment:
we don't want to copy the source file.  we want to copy in the built elasticsearh_logging_discovery binary that gets built in the Makefile from the `binary` directive.

Reply:
ah... sorry, my fault...